app/scripts/metamask-controller.js

```diff
@@ -345,14 +345,14 @@ export default class MetamaskController extends EventEmitter {
           this.assetsContractController.getERC1155TokenURI.bind(
             this.assetsContractController,
           ),
-        onCollectibleAdded: ({ address, symbol, tokenId, standard, source }) =>
+        onNftAdded: ({ address, symbol, tokenId, standard, source }) =>
           this.metaMetricsController.trackEvent({
             event: EVENT_NAMES.NFT_ADDED,
```

Comment:
@Gudahtt can you change the event name from `EVENT_NAMES.NFT_ADDED ` to `EVENT_NAMES.TOKEN_ADDED`?  Despite `NFT_ADDED` being defined in `constants/metametrics.js` we've never actually used it yet and this would be the preferred way. (ie one event capturing the generalized activity and specific properties to split it out if needed)

Reply:
Sounds reasonable! I might do this in a separate PR though, it doesn't seem directly related to the controllers update. This code is pre-existing (possibly still behind a build flag)